plugin-guide.md

```diff
@@ -83,17 +125,30 @@ workflow engine's identifier means. That is somewhat the responsibility of the
 system administrator.
 
 The class `BaseJsonRuntimeProvisioner` is a partial implementation that can
-store crash recovery information in a JSON object of the implementor's
-choosing,
-making recovery easier.
+store crash recovery information in a JSON object of the implementer's
+choosing, making recovery easier.
 
 # Workflow Engine
 Workflow engines implement `ca.on.oicr.gsi.vidarr.WorkflowEngineProvider`
 and `ca.on.oicr.gsi.vidarr.WorkflowEngine`. These plugins are responsible for
-running workflows and collecting the output from the workflow.
+running workflows and collecting the output from the workflow. A workflow
+engine can support multiple languages (see
+`ca.on.oicr.gsi.vidarr.WorkflowLanguage` for a complete list) and choose which
+subset it can accommodate.
+
+The workflow engine will be given the complete input to the workflow (with real
+paths provided by the input provisioners) and the workflow itself. Once the
+workflow has completed, it must provide a JSON structure that references the
+output of the workflow. Output files generated by the workflow engine will be
+passed to the output provisioners.
```

Comment:
Would appreciate clarification that 'will be passed' indicates that Vidarr will do this without the plugin developer needing to do anything (unless I'm mistaken)